### PR TITLE
#164004452 Implement chore for global font and styling

### PR DESCRIPTION
**Pull request by iAfolayan**
### What does this PR does?
- Implement global font and styling

https://www.pivotaltracker.com/story/show/164004452

### CODE REVIEW

File: src/styles/partials/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+a {
+  color: $primarycolor;
+}
+
+a:hover {
+  color: $black;
+}
+
+/**** PROFILE PAGE *****/
```
Comment (codrex) [inline src/styles/partials/_styles.scss]: Why do we have this here

Comment (iAfolayan) [reply src/styles/partials/_styles.scss]: The profile page comment is to indicate where the global styling for profile pages commence.

Comment (codrex) [reply src/styles/partials/_styles.scss]: why do we have global styles for profile page?

Comment (codrex) [reply src/styles/partials/_styles.scss]: @iAfolayan ☝️

**Review by codrex — APPROVED** [review]

**Pull request merged**